docs/troubleshooting.md

```diff
@@ -1,5 +1,65 @@
 # Troubleshooting
 
+## Recover UI password
+
+The first time you access your DAppnode using the VPN, appear the next image:
+
+![Signin](./images/sign_in_image.png)
+
+Once you fill the fields and press the register button, DAppNode provides you with a recovery code. If you forget te DAppNode's Password you will be able to access using this token.
+
+![Recovery Token](./images/recovery_token.png)
+
+You should save your token in a secure place like a password manager, paper, etc. In this case, we would save this:
+
+~~~
+4LMB9w3l50Yljwr6bIgQ
+~~~
+
+### How to access to my DAppNode using the recovery token
+
+Firstly, we have to go to the login page of our DAppNode:
+
+![login](./images/login.png "")
+
+We should select the option "Forgot password?". It will appear a field where you can reset your password using the recovery token. In this example, we would use the token <code>4LMB9w3l50Yljwr6bIgQ</code>. You have to use your recovery token.
+
+![reset password with the recovery token](./images/reset_password_with_recovery_token.png "")
+
+After confirming the recovery token, you can define an admin user in DAppNode and his password in the same way you did the first time you connect.
+
+![Defining the new password](./images/sign_in_image.png "")
+
+Remember using a password with the next properties:
+
+* Minimum 8 characters
+* At least one Cap letter
+* At least one number
+
+Once you filled the field and press the register button, dappnode will provide a recovery token.
+
+![Obtaining your recovery token](./images/recovery_token.png "")
+
+We write down and save that token and we accept we have copied our recovery token. We log in with our new credentials and we have access to our DAppNode.
+
+### How to recover your recovery token and recover admin access
+
+In case we have lost bot the password and the recovery token, we need to access our DAppNode: SSH into the DAppNode host or plug a keyboard.
```

Comment:
```suggestion
In case we have lost both the password and the recovery token, we need to access our DAppNode: SSH into the DAppNode host or plug a keyboard.
```